service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java

```diff
@@ -74,15 +103,23 @@ private Workflow sendRequest(Workflow workflow, String requestPath, String tenan
         logger.info("Workflow is actvie = {}, deploymentID = {}", responseWorkflow.isActive(), deploymentId);
         return workflowRepo.save(responseWorkflow);
       } catch (Exception e) {
-        throw new WorkflowEngineServiceException("Unable to get updated workflow from workflow engine!");
+
```

Comment:
Should this have a `logger.debug()` call in the exception?

Reply:
https://github.com/TAMULib/mod-workflow/blob/0b23024970be2453c72ee1d8f98baf602d97c351/service/src/main/java/org/folio/rest/workflow/controller/advice/WorkflowControllerAdvice.java#L26